.github/workflows/test.yml

```diff
@@ -48,19 +48,14 @@ jobs:
       github.repository == 'mvdan/sh'
     env:
       # Export environment variables for all stages.
-      DOCKER_CLI_EXPERIMENTAL: enabled # for 'docker buildx'
-      DOCKER_USER: mvdan
-      DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+      DOCKER_USER: ${{ secrets.DOCKER_USER }}
+      # Pushing READMEs to Dockerhub currently only works with username/password
+      # and not with personal access tokens (Step: Update DockerHub description)
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       DOCKER_REPO: shfmt
       # We use all platforms for which FROM images in our Dockerfile are
       # available.
-      DOCKER_PLATFORMS: >
-        linux/386
-        linux/amd64
-        linux/arm/v7
-        linux/arm64/v8
-        linux/ppc64le
-
+      DOCKER_PLATFORMS: linux/386,linux/amd64,linux/arm/v7,linux/arm64/v8,linux/ppc64le
```

Comment:
I found the older syntax easier to maintain to be honest; adding or deleting an item would just affect that line, and it was easy to review. I imagine this list will get longer, too.

Reply:
Agree, I also prefer the old syntax, but this seems how it should be used: https://github.com/docker/build-push-action#multi-platform-image

I can check if the following could work if you want:
```yaml
  DOCKER_PLATFORMS: |
    linux/386,
    linux/amd64,
    linux/arm/v7,
    linux/arm64/v8,
    linux/ppc64le
```

Reply:
The way that the previous version of the code did that is by programmatically converting the multi-line (space-separated) list into the right comma-separated format `--platform ${DOCKER_PLATFORMS// /,}`. I'm not sure if you can still do that directly where you need it, since `platforms: ${{ env.DOCKER_PLATFORMS }}` probably doesn't allow shell expansion. Alternatively, you could make it part of a `run` step and put it into a separate environment variable, like you do e.g. for setting up `$DOCKER_BASE`.

Reply:
Sure, and we already have a run that defines some env variables (`Set up image tag`) so, I am happy with both solutions, let me know @mvdan what you prefer.

Reply:
I do prefer keeping the newline-separated list, if it is just one extra line of shell :)

Reply:
ack